oxide-vpc/src/engine/overlay.rs

```diff
@@ -428,6 +430,25 @@ impl VpcMappings {
         }
     }
 
+    /// Delete the mapping for the given VIP in the given VNI.
+    ///
+    /// Return the existing entry, if there is one.
+    pub fn del(&self, vip: &IpAddr, phys: PhysNet) -> Option<PhysNet> {
```

Comment:
I believe this type is `Copy`, but me might want to take this by reference. The method doesn't really need to take ownership of that, and would be nice to signal in the interface.

Reply:
Yep, meant to do that, thanks for catching.